storey/utils.py

```diff
@@ -42,6 +42,12 @@ def get_one_unit_of_duration(string_time):
     return multiplier
 
 
+def get_window_start_time_by_time(timestamp, window_millis):
```

Comment:
I think you can just inline this one – doesn't seem like a utility for anything other than aggregations.

Reply:
@gtopper 
function removed and code is moved to its caller